src/commands/statusline.ts

```
@@ -100,14 +100,15 @@ export const statuslineCommand = define({
 		},
 		config: sharedArgs.config,
+		debug: sharedArgs.debug,
 	},
 	async run(ctx) {
 		// Set logger to silent for statusline output
 		logger.level = 0;
 
 		// Load configuration and merge with CLI args
-		const config = loadConfig(ctx.values.config);
-		const mergedOptions = mergeConfigWithArgs(ctx, config);
+		const config = loadConfig(ctx.values.config, Boolean(ctx.values.debug));
```

ESLint

[error] Unsafe argument of type `any` assigned to a parameter of type `string | undefined`.

(ts/no-unsafe-argument)